forge/src/multi_runner.rs

```diff
@@ -173,14 +165,12 @@ pub struct MultiContractRunner {
     pub contracts: BTreeMap<String, (Abi, ethers::prelude::Bytes, Vec<ethers::prelude::Bytes>)>,
     /// Compiled contracts by name that have an Abi and runtime bytecode
     pub known_contracts: BTreeMap<String, (Abi, Vec<u8>)>,
-    /// Identified contracts by test
-    pub identified_contracts: BTreeMap<String, BTreeMap<Address, (String, Abi)>>,
     /// The EVM instance used in the test runner
     pub evm_opts: EvmOpts,
     /// The EVM spec
     pub evm_spec: SpecId,
-    /// All contract execution info, (functions, events, errors)
-    pub execution_info: (BTreeMap<[u8; 4], Function>, BTreeMap<H256, Event>, Abi),
+    /// All known errors, used for decoding reverts
+    pub errors: Option<Abi>,
```

Comment:
We now only need to pass around an abi with all the known errors we have. The reason this is passed around is only because of `call`/`call_committing` which decodes reverts. I think we should refactor them out at some point and just decode the reverts later